orangecontrib/network/network/embeddings.py

```diff
@@ -131,8 +133,16 @@ def __call__(self, G):
         walks = self._simulate_walks(G)
         walks = [list(map(str, walk)) for walk in walks]
 
-        model = Word2Vec(walks, size=self.emb_size, window=self.window_size, min_count=0, sg=1, workers=4,
-                         iter=self.num_epochs, callbacks=self.callbacks)
+        # gensim changed "size" param to "vector_size" in v. 4.0.0
+        # https://github.com/RaRe-Technologies/gensim/wiki/Migrating-from-Gensim-3.x-to-4
+        params = {"window": self.window_size, "min_count": 0, "sg": 1,
+                  "workers": 4, "iter": self.num_epochs, "callbacks":
+                  self.callbacks}
```

Comment:
Hint: no need to change anything, but `params = dict(window=self.window_size, min_count=0, sg=1, ...)` would save a lot of quotes.